pkg/inttest/k8s.go

```diff
@@ -46,3 +52,43 @@ type K8sClient struct {
 	Client *kubernetes.Clientset
 	Config []byte
 }
+
+func (k K8sClient) AssertPodIsReady(t *testing.T, group string, instance string) {
```

Comment:
Please add a little godoc maybe stating that it times out after 20 seconds.

Reply:
I was thinking about passing the timeout as a parameter since it might differ depending on the instance we're deploying